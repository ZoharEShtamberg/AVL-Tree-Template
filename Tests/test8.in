add_country 1 0
add_country 1 0
add_team 31 31 2
add_team 11 1 1
add_contestant 1 1 1 800
add_contestant 2 1 1 700
add_contestant 3 1 1 600
add_contestant 4 1 1 500
add_contestant 10 1 1 80
add_contestant 20 1 1 70
add_contestant 30 1 1 60
add_contestant 40 1 1 50
add_contestant 100 1 1 8
add_contestant 200 1 1 7
add_contestant 300 1 1 6
add_contestant 400 1 1 5
add_contestant 5 1 1 400
add_contestant 6 1 1 300
add_contestant 7 1 1 200
add_contestant 50 1 1 40
add_contestant 60 1 1 30
add_contestant 70 1 1 20
add_contestant 500 1 1 4
add_contestant 600 1 1 3
add_contestant 700 1 1 2
add_contestant_to_team 11 1
add_contestant_to_team 11 2
add_contestant_to_team 11 3
add_contestant_to_team 11 4
add_contestant_to_team 11 10
add_contestant_to_team 11 20
add_contestant_to_team 11 30
add_contestant_to_team 11 40
add_contestant_to_team 11 100
add_contestant_to_team 11 200
add_contestant_to_team 11 300
add_contestant_to_team 11 400
add_contestant_to_team 11 5
add_contestant_to_team 11 6
add_contestant_to_team 11 7
add_contestant_to_team 11 50
add_contestant_to_team 11 60
add_contestant_to_team 11 70
add_contestant_to_team 11 500
add_contestant_to_team 11 600
add_contestant_to_team 11 700
remove_contestant_from_team 11 500
remove_contestant_from_team 11 600
remove_contestant_from_team 11 700
get_team_strength 11
add_country 2 0
add_team 22 2 1
add_contestant 34 2 1 1
add_contestant 35 2 1 1
add_contestant 36 2 1 1
add_contestant_to_team 22 34
add_contestant_to_team 22 35
add_contestant_to_team 22 36
play_match 11 22
get_medals 1
get_medals 2