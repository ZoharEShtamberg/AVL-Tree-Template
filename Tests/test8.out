add_country: SUCCESS
add_country: FAILURE
add_team: FAILURE
add_team: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
remove_contestant_from_team: SUCCESS
get_team_strength: SUCCESS, 1030
add_country: SUCCESS
add_team: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
add_contestant_to_team: SUCCESS
play_match: SUCCESS
get_medals: SUCCESS, 1
get_medals: SUCCESS, 0
